docusaurus/docs/React/contexts/channel-state-context.mdx

```diff
@@ -67,20 +67,37 @@ Error object (if any) in loading the `channel`, otherwise null.
 
 ### giphyVersion
 
-The giphy version to use when displaying giphies. 
+The giphy version to use when displaying giphies.
 
 | Type                                                                                                                                                |
 | --------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `'original' | 'fixed_height' | 'fixed_height_still' | 'fixed_height_downsampled' | 'fixed_width' | 'fixed_width_still' | 'fixed_width_downsampled'` |
 
 ### hasMore
 
-If the channel has more messages to paginate through.
+If the channel has more, older, messages to paginate through.
```

Comment:
You can paginate both ways - considering the fact that you can jump to older messages now.
```suggestion
If the channel has more messages to paginate through.
```